src/main.cc

```diff
@@ -236,20 +240,16 @@ For more information visit " DESC_MANUFACTURER_URL "\n\n");
             case 10: // --version
                 print_version = true;
                 break;
+            case 12: // --create-config
+                create_config = true;
+                break;
             }
             break;
         default:
             break;
         }
     }
 
-    if (print_version) {
```

Comment:
Moved this **below** the `--create-config` execution.  This allows `--create-config` to generate cleanly.